dbms/src/Flash/Planner/PhysicalPlanNode.h

```diff
@@ -70,8 +78,11 @@ class PhysicalPlanNode
 
     String toString();
 
+    // Release all references to child nodes.
```

Comment:
This doesn't seem necessary. For a plan tree, there is no possibility that a circular dependency exists between nodes.

Reply:
ok, removed.